pkg/controllermanager/apis/config/v1alpha1/types.go

```diff
@@ -95,6 +98,17 @@ type ControllerRegistrationControllerConfiguration struct {
 	ConcurrentSyncs int `json:"concurrentSyncs"`
 }
 
+// EventControllerConfiguration defines the configuration of the Event controller.
+type EventControllerConfiguration struct {
+	// ConcurrentSyncs is the number of workers used for the controller to work on
+	// events (defaults to 5).
+	// +optional
+	ConcurrentSyncs *int `json:"concurrentSyncs,omitempty"`
```

Comment:
```suggestion
	ConcurrentSyncs int `json:"concurrentSyncs"`
```
Same as previous comment.